app/views/govuk_publishing_components/components/docs/skip_link.yml

```diff
@@ -0,0 +1,11 @@
+name: Skip link
+description: Skip link component helps keyboard-only users skip to the main content on a page
+accessibility_criteria: |
+  - the link is the first items that screen readers hear and that keyboard users tab to
+  - is becomes visible when pressing tab
+shared_accessibility_criteria:
+- link
```

Comment:
Not all link criteria are relevant, (the link doesn't have visible text, for example, until it's in focus) and the component IS a link, rather than a container for links. I'd pick and choose the relevant ones and put them in criteria and leave out the shared ones.

Reply:
Updated as suggested (dropped the shared criteria an rewrite explicitly